auth/client-api.md

```diff
@@ -14,7 +14,7 @@ abstract class Client {
   public abstract request(params: RequestParams): Promise<{ uri, id }>;
 
   // respond wallet authentication
-  public abstract respond(params: RespondParams): Promise<boolean>;
+  public abstract respond(result: Resutl<RespondParams, Error>): Promise<boolean>;
```

Comment:
```suggestion
  public abstract respond(result: Result<RespondParams, Error>): Promise<boolean>;
```

Reply:
didn't ask for review yet 🤣 
I think it is not javascript btw

Reply:
but commited xd